src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -151,19 +152,20 @@ void CHyprGroupBarDecoration::draw(CMonitor* pMonitor, float a) {
         CColor            color = m_dwGroupMembers[WINDOWINDEX].lock() == g_pCompositor->m_pLastWindow.lock() ? PCOLACTIVE->m_vColors[0] : PCOLINACTIVE->m_vColors[0];
         color.a *= a;
 
+        if (*PBOTTOMBAR)
```

Comment:
the thin bottom bar looks bad with rounding, I added an option to disable the thin bottom bar, rounding the thin bar is tricky because the height is very small and sometimes opengl throws an error for rendering a negative height.

A problem that arises because of this is that the gap between the main group bar and the window is kinda awkward without the thin bar, I want the extra space to be taken up by the application open and am open to suggestions on how to implement this.

Reply:
like I said before rounding has to be less or equal to max(width/2, height/2)

to remove the space from the indicator  set `BAR_INDICATOR_HEIGHT` to zero wherever it appears)
specially in `const auto ONEBARHEIGHT = BAR_PADDING_OUTER_VERT + BAR_INDICATOR_HEIGHT + (*PGRADIENTS || *PRENDERTITLES ? *PHEIGHT : 0);` 

I am not sure there is a point to the thin bar anymore maybe it can be removed altogether, but ask vaxry